### DISCUSSION TITLE
打包白屏问题求助，感谢感谢

### DISCUSSION THREAD

**Discussion by zhangchengqiang** [umijs/umi#11297] [Q&A] [answered]
我需要部署到远程二级域名下,进行以下配置
`  outputPath: './app',
  publicPath: "/app/",`
本地umi run dev 使用正常，
umi build以后，在app文件夹启动sever，可以在index.html页面找到资源umi.js,umi.css

![sever启动效果图](https://github.com/umijs/umi/assets/25154964/bc31ceae-7ac9-4904-ad1c-17fd817dae94)

但是出现白屏情况

**Comment by fz6m** [comment, marked as answer]
没匹配上路由，需要配置 `base: '/app'`

**Reply by zhangchengqiang** [reply]
多谢多谢，感激不尽感激不尽
